src/sentry/relay/config/metric_extraction.py

```diff
@@ -120,6 +122,25 @@ def get_metric_extraction_config(
     return {
         "version": _METRIC_EXTRACTION_VERSION,
         "metrics": metric_specs,
+        "extrapolate": extrapolation_config,
+    }
+
+
+def get_extrapolation_config(project):
+    if not features.has("organizations:metrics-extrapolation", project.organization):
+        return {}
+
+    enabled = project.get_option("sentry:extrapolate_metrics", None)
+    if enabled is None:
+        enabled = project.organization.get_option("sentry:extrapolate_metrics", False)
+    if not enabled:
+        return {}
```

Comment:
This assumes fallback/override logic between the project and org option: If the project option is not set or explicitly set to `None`, the organization option is used. The settings UI can offer an "inherit organization setting" choice.